app/views/templates/player.jst

```diff
@@ -0,0 +1,49 @@
+<audio id="player" autoplay preload autobuffer></audio>
+<div class="player">
+    <div class="play-controls">
+        <% if (false) { %>
+            <img class="album" src="album"></img>
+        <% } else { %>
+            <svg class="album">
+                <use xlink:href="<%= require('../../assets/icons/music-lg.svg') %>"/>
+            </svg>
+        <% }%>
+
+        <svg id="prev">
+            <use xlink:href="<%= require('../../assets/icons/prev-lg.svg') %>"/>
+        </svg>
+        <svg id="play">
+            <use xlink:href="<%= require('../../assets/icons/play-lg.svg') %>"/>
+        </svg>
+        <svg id="next">
+            <use xlink:href="<%= require('../../assets/icons/next-lg.svg') %>"/>
+        </svg>
+    </div>
+    <div class="timeline">
+        <div class="text-container">
+            <p id="trackname">Pas de musique :(</p>
+        </div>
+        <div id="progress-bar">
```

Comment:
the `progress` element is hard to theme correctly in all browser...
i think we can leave it as is.